crates/tabby/src/services/completion.rs

```diff
@@ -321,35 +322,51 @@ impl CompletionService {
             self.config.max_decoding_tokens,
         );
 
+        let mut use_crlf = false;
         let (prompt, segments, snippets) = if let Some(prompt) = request.raw_prompt() {
             (prompt, None, vec![])
         } else if let Some(segments) = request.segments.as_ref() {
+            let mut new_segments = segments.clone();
+            if segments.prefix.contains("\r\n") {
```

Comment:
what if prefix contains both `\r\n` and `\n`? what's the strategy here?

Reply:
normally, a file should not contain both `\r\n` and `\n` in the same time?

IMO, just replaced all CRLF to LF, and the same logic for output